stages/org.osbuild.sysconfig

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/python3
+"""
+
+Configure sysconfig files
+
+The sysconfig directory contains a variety of system configuration files. The
+values found in each of the configuration files may need to be specified
+before first boot.
+
+Currently, only the kernel and network configs can be modified.
+
+"""
+
+import sys
+import os
+
+import osbuild.api
+
+
+SCHEMA = """
+"additionalProperties": false,
+"properties": {
+  "kernel": {
+    "additionalProperties": false,
+    "type": "object",
+    "description": "sysconfig kernel options",
+    "properties": {
+      "update_default": {
+        "type": "boolean",
+        "description": "This option makes a newly installed kernel as the default in the boot entry selection."
+      },
+      "default_kernel": {
+        "type": "string",
+        "description": "This option specifies what package type will be used as the default."
+      }
+    }
+  },
+  "network": {
+    "additionalProperties": false,
+    "type": "object",
+    "description": "sysconfig network options",
+    "properties": {
+      "networking": {
+        "type": "boolean",
+        "description": "Enables or disables networking"
+      },
+      "no_zero_conf": {
+        "type": "boolean",
+        "description": "Disables the zero configuration network suite"
+      }
+    }
+  }
+}
+"""
+
+# sysconfig uses yes and no instead of true and false
+def bool_to_string(bool):
+  if bool:
+    return "yes"
+  else:
+    return "no"
+
+
+def configure_kernel(tree, kernel_options):
+  with open(f"{tree}/etc/sysconfig/kernel", 'w') as kernel_file:
+    for option, value in kernel_options.items():
+      if option == "update_default":
+        kernel_file.write(f"UPDATEDEFAULT={bool_to_string(value)}\n")
+      if option == "default_kernel":
+        kernel_file.write(f"DEFAULTKERNEL={value}\n")
+
+  return 0
```

Comment:
Not really needed, since you are also not checking that ;)

Reply:
Sorry. I updated the lines. What isn't really needed?